docs/api-analytics.yml

```diff
@@ -76,6 +76,26 @@ paths:
           $ref: "#/components/responses/NoResults"
         "500":
           $ref: "#/components/responses/InternalError"
+  /api/analytics/v2/ledger/richlist:
+    get:
+      tags:
+        - ledger
+      summary: Returns a richlist.
```

Comment:
Reminder.

Reply:
Reminder for what?

Reply:
Renaming `richlist` to `richest-addresses` :trollface: